src/components/SettingsWheel/SettingsWheel.css

```diff
@@ -1,22 +1,31 @@
 .SettingsWheel {
 	width: 50px;
 	height: 50px;
-	background: url(./SettingsWheel.png) no-repeat;
+	background: url(./SettingsWheel.svg) no-repeat;
 	border: none;
+	height: 50px;
+	position: absolute;
+	top: 15px;
 	transition: transform 0.3s ease-in-out;
 	transform: scale(1);
+	right: 24px;
+	width: 50px;
 }
+
 .SettingsWheel:hover {
 	cursor: pointer;
 	transform: rotate(90deg);
 }
+
 .SettingsWheel:focus {
-	transform: scale(1.1);
+	transform: scale(1);
```

Comment:
Does ``transform: scale(1)`` ever change value now? It may be worth removing this line and the one above

Reply:
Removed the scale